<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TD6_Class</class>
 <widget class="QMainWindow" name="TD6_Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TD6_</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="descriptionLineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>261</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="prixLineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ajouterButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="retirerButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Retirer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reinitialiserButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Reinitialiser</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter taxes</string>
    </property>
   </widget>
   <widget class="QListWidget" name="articlesList">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>321</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="sanstaxes">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>270</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SOUS-TOTAL :</string>
    </property>
   </widget>
   <widget class="QLabel" name="taxes">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>290</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TAXES (TPS/TVQ) :</string>
    </property>
   </widget>
   <widget class="QLabel" name="total">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>310</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TOTAL :</string>
    </property>
   </widget>
   <widget class="QLabel" name="produit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Produit : </string>
    </property>
   </widget>
   <widget class="QLabel" name="prix">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>49</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Prix : </string>
    </property>
   </widget>
   <widget class="QLabel" name="totalAvantTaxesLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>270</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="totalTaxesLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>290</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="totalAPayerLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>310</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="affichage.qrc"/>
 </resources>
 <connections/>
</ui>
